x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_container.js

```diff
@@ -104,7 +104,7 @@ function ExplorerChartContainer({ series, severity, tooManyBuckets, wrapLabel })
             <EuiToolTip position="top" content={textViewButton}>
               <EuiButtonEmpty
                 iconSide="right"
-                iconType="stats"
+                iconType="visLine"
```

Comment:
Could you also switch the actions menu for the anomalies table to use `visLine`:
![image](https://user-images.githubusercontent.com/7405507/90526216-041bca80-e168-11ea-96ee-42fea4729c77.png)

Reply:
Fixed in a54d94f.